File: pe-04/practice.py
score = 85
missed_sessions = 0
asked_for_help = False

status = ""

if score >= 80 and missed_sessions <= 1:
    status = "strong"
elif score < 60 or missed_sessions > 1:
    status = "follow-up"
else:
    status = "developing"

print(status)

File: pe-04/practice.ts
export {};
const score: number = 85;
const missedSessions: number = 0;
const askedForHelp: boolean = false;

let status: string;

if (score >= 80 && missedSessions <= 1) {
    status = "strong";
} else if (score < 60 || missedSessions > 1) {
    status = "follow-up";
} else {
    status = "developing";
}

console.log(status);